canonicalwebteam/blog/wordpress_api.py

```diff
@@ -8,7 +8,7 @@
 )
 
 
-api_session = CachedSession(fallback_cache_duration=3600)
+api_session = CachedSession(fallback_cache_duration=3600, timeout=(1, 100))
```

Comment:
why increasing to `100` this is a lot, please increase to `0.5, 5` if you think the payloads are too big

Reply:
Development artifact. Removed :)